docs/docker.md

```diff
@@ -133,6 +132,83 @@ docker images and good practice should be followed:
 
 -   Additional alpine-based builds may be requested for some components.
 
+### Label description content
+
+Dockerfiles **SHOULD** put the following labels inside the Dockerfile:
+
+- `maintainer` - the name of the caretaker organization.
+- `gitrepo` - where the source code can be found.
+- `branch` - the branch the image was built from.
+- `release_date` - the date the image was built.
+- `contacts` - comma separated partial email addresses of responsible contacts (e.g for security updates)
+- `organization` - partial email address(es) of responsible contacts (to avoid full public addresses / spam)
+
+Keep in mind that could be possible predefine some values for these labels and modify then during the
+building of the docker image. In the following example, we have predefined some values in the Dockerfile
+content and during the construction of the docker image, we modify some of them. The reason is to allow the
+building outside of Docker Hub or without specifing the corresponding arguments in the command line. Note 
+that a Docker build hook can be used to automate the date and branch functions.
```

Comment:
Actually anything that can go in the build script (e.g. date) or anything that is predefined by Docker Hub (assuming automated builds have been set up.)

Reply:
done